package.json

```diff
@@ -121,5 +121,9 @@
     "*.yml": [
       "yarn format-staged"
     ]
+  },
+  "engines": {
+    "node": ">=16.0.0 <17.0.0",
+    "npm": ">=8.0.0 <9.0.0"
```

Comment:
Do we need this? I think we use `yarn` mostly, no?

Reply:
ooh yeah. I'll update that